packages/cli/src/commands/alias/index.ts

```diff
@@ -38,6 +38,7 @@ const help = () => {
     -S, --scope                         Set a custom scope
     -N, --next                          Show next page of results
     -y, --yes                           Skip the confirmation prompt when removing an alias
+    --limit                             Number of results to return per page
```

Comment:
**suggestion (non-blocking):** add default and max

Should we add the default and max limit values for each of these commands? We could say:

```suggestion
    --limit                             Number of results to return per page (default 20, max 100)
```

Reply:
Good idea, thanks!